src/components/navbar/ReaderNavBar.tsx

```diff
@@ -261,6 +261,7 @@ export default function ReaderNavBar(props: IProps) {
                             showPageNumber={settings.showPageNumber}
                             loadNextOnEnding={settings.loadNextOnEnding}
                             skipDupChapters={settings.skipDupChapters}
+                            fitToWindow={settings.fitToWindow}
```

Comment:
I'd rename it to e.g. `fitPageToWindow`, to make it 100% clear

Reply:
agree, fixed in next commit